### ISSUE TITLE
[SOURCE] Google Assistant

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/pipedream#804]
**Describe the event source. What app, what event?**

TBD - need to research what would be the most useful.

**Please provide a link to the relevant API docs for the specific service / operation this trigger is tied to.**

https://assistant.google.com/

**Comment by vellames** [comment]
Hey @dylburger . Do we have an app for Google Assistant? I cant see it

**Comment by dylburger** [comment]
It doesn't look like it. I created another ticket and will mark this as blocked. Can you remove from the Component Dev board for now?

**Comment by dylburger** [comment]
Blocked on #4174

**Comment by malexanderlim** [comment]
This API has been deprecated, closing out this issue.
https://developers.google.com/assistant/ca-sunset
